### PR TITLE
Portfolio main page & styles for mobile devices

### PR DESCRIPTION
**Pull request by jr-cast**
What was added to the project:

- Main index and CSS files were added in the first iteration with a simpler layout of the selected template from the Figma website.
- Files were modified to address mobile devices from the get-go on the 2nd iteration
- However, Mobile display work of the document was placed on hold, it was decided that the main page and styles must be fully addressed for regular PC screens
- Refactoring and completing the regular display of the document
- Finally, mobile and responsiveness tasks were  resumed

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        <header>
+            <nav>
+                <ul>
+                    <li><span id="port"><a href="">Portfolio</a></span></li>
+                    <li><span id="about"><a href="">About</a></span></li>
+                    <li><span id="contact"><a href="">Contact</a></span></li>
+                </ul>
+            </nav>
+            <div class="burguerMenu">
+                <span>Welcome</span>
```
Comment (sja-thedude) [inline index.html:27]: - [ ] Kindly wrap the logo `Welcome` in `a tag`.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                <div></div>
+                <i class="fa-solid fa-bars fa-2x"></i>
```
Comment (sja-thedude) [inline index.html:29]: - [ ] Kindly wrap the hamburger button in `nav tag`.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                don’t hesitate to contact me.</p>
+        </section>
+        <footer>
+            <div class="social">
+                <a href="#"><i class="fa-brands fa-github fa-lg"></i></a>
+                <a href="#"><i class="fa-brands fa-linkedin fa-lg"></i></a>
+                <a href="#"><i class="fa-brands fa-whatsapp-square fa-lg"></i></a>
+                <a href="#"><i class="fa-solid fa-envelope-open "></i></a>
+                <a href=""><i class="fa-solid fa-square-poll-vertical fa-lg"></i></a>
+            </div>
```
Comment (sja-thedude) [inline index.html:52]: - [ ] Kindly wrap the social icons in` ul tag` to group them instead of `div`.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+                like what you see and have a project you need coded,
+                don’t hesitate to contact me.</p>
+        </section>
+        <footer>
+            <div class="social">
+                <a href="#"><i class="fa-brands fa-github fa-lg"></i></a>
+                <a href="#"><i class="fa-brands fa-linkedin fa-lg"></i></a>
+                <a href="#"><i class="fa-brands fa-whatsapp-square fa-lg"></i></a>
+                <a href="#"><i class="fa-solid fa-envelope-open "></i></a>
+                <a href=""><i class="fa-solid fa-square-poll-vertical fa-lg"></i></a>
```
Comment (sja-thedude) [inline index.html:51]: - [ ] Make sure to wrap each icon in `li tag` as it should be inside `ul tag`.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+            </div>
+
+            <i id="drop" class="fa-solid fa-circle-chevron-down fa-2x"></i>
```
Comment (sja-thedude) [inline index.html:54]: - [ ] Make sure to wrap the drop-down button in a `button tag` as its a button.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 98vh;
+  text-align: center;
+}
+
+body {
+  font-family: 'Inter', sans-serif;
+  background-image: url(/static/Header-llustration-desktop@2x.png);
```
Comment (sja-thedude) [inline style.css:99]: - [ ] Kindly fix the background image, it is not displayed on the web.
- [ ] Also fix the background colour which should be white.
- Your design:
Screenshot 2022-03-17 at 8 56 34 AM

- Figma design:
Screenshot 2022-03-17 at 8 55 55 AM

File: style.css [hunk 7]
```diff
@@ -0,0 +1,218 @@
...
+  background-size: cover;
+  background-color: #e5e5e5;
+  background-repeat: no-repeat;
+}
+
+#drop {
+  padding-top: 50px;
+}
```
Comment (sja-thedude) [inline style.css:107]: - Kindly make sure to follow the button colour as per the Figma design.
Your design:
Screenshot 2022-03-17 at 9 22 52 AM

Figma design:
Screenshot 2022-03-17 at 9 22 22 AM

File: style.css [hunk 8]
```diff
@@ -0,0 +1,218 @@
...
+h1,
+h2 {
+  font-size: 48px;
+  margin: 0;
+  padding: 0;
+  font-style: normal;
+  font-weight: 800;
+  line-height: 60px;
+  letter-spacing: 0.37px;
+}
```
Comment (sja-thedude) [inline style.css:67]: - [ ] Make sure to follow the font family and spacing between elements as per the Figma design.
- Your design:
Screenshot 2022-03-17 at 8 56 34 AM
- Figma design:
Screenshot 2022-03-17 at 8 55 55 AM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @jr-cast,  This is @sja-thedude your code reviewer.

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Professional readme 👍 
- Good PR title and description 👍 
- Nice work on implementing additional changes by adding the desktop version as well 👍 
- All Linters are passing.

## Required
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jr-cast** [comment]
Modifications were addressed. 

- Apparently, the background image was not being displayed correctly (although locally in my computer it was showing).

File: index.html [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+                don’t hesitate to contact me.</p>
+        </section>
+        <footer>
+            <ul class="social">
+                <li><a href="#"><i class="fa-brands fa-github fa-lg"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-linkedin fa-lg"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-whatsapp-square fa-lg"></i></a></li>
+                <li><a href="#"><i class="fa-solid fa-envelope-open "></i></a></li>
+                <li><a href=""><i class="fa-solid fa-square-poll-vertical fa-lg"></i></a></li>
+            </ul>
```
Comment (CollinsTatang) [inline index.html:53]: - [ ] Kindly fix the social media icons and adjust the paragraph text padding according to [figmal design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
### **See the Error**
socialMedia

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @jr-cast, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
 

- [ ] Please try to complete all the changes requested by the previous code reviewer.

 
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 9 above)
Comment (jr-cast) [reply index.html:53]: Thanks for the comments, with fixing the social media icons and adjusting the paragraph text padding you mean to just avoid overlapping right? Thanks

Comment (CollinsTatang) [reply index.html:53]: - [ ] Please follow this design 

see

**Comment by jr-cast** [comment]
Overlapping was addressed:

image

**Review by AmaduKamara — APPROVED** [review]
# STATUS: APPROVED 🟢🟢🟢

Hi @jr-cast  

Good job implementing all the changes asked by the previous reviewers.
Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**